app.js

```diff
@@ -0,0 +1,96 @@
+// question 1
+
+// for a light switch statement you would use a if else statemet
+var light = 'bright'
+if(switchup === on){
+	return bright
+} else(swithDown === off){
+	return dark
+}
+
+//question 2
+// for a email you would use object 
+var user {
+	email: 'dare'
+	password: 'ginsing'
+
+}
+
+//question 3
+// you wourld make an variables
+var spaceship = ['hull', 'laserblaster', 'tractor beam', 'warp drive']
```

Comment:
you would use an object... or maybe an array

+0.5 because you used an array, although you didn't not explicitly answer the question